### DISCUSSION TITLE
Is it possible to build musl-libc based OS & glibc based OS with one file?

### DISCUSSION THREAD

**Discussion by typenoob** [pyinstaller/pyinstaller#8027] [Features & Development]
Hi! I built a file in alpine which could be properly run on musl-libc based OS like openwrt or apline.But it couldn't run on ubuntu which is based on glibc .Is there any elegant approach to build one file for these two kinds of platforms?If not, Is there any plan to support it in the future?

**Comment by bwoodsend** [comment]
You can try using [staticx](https://staticx.readthedocs.io/en/latest/usage.html) but just building two applications is a far more sensible approach.
